docs/make_projection.py

```diff
@@ -60,9 +64,20 @@ def find_projections():
                          'crs', 'projections.rst')
     table = open(fname, 'w')
 
-    table.write('.. _cartopy_projections:\n\n')
-    table.write('Cartopy projection list\n')
-    table.write('=======================\n\n\n')
+    notes = \
+        """             .. (comment): DO NOT EDIT this file.
```

Comment:
Technically, you don't need all this space at the beginning, since you strip each line below.